test/test_generator.py

```diff
@@ -213,3 +215,6 @@ def test_code_generator_block_gen(self):
     def test_generator_cmdline(self):
         t = TesterCmdline(None, ['-v'])
         self.assertTrue(t.args.verbose)
+
+if __name__ == '__main__':
```

Comment:
One can still use `pytest` or whatever. This allows individual file execution without additional dependencies. It's a quick-n-dirty way for anyone to execute a single suite of tests in a particular file. It's not uncommon as far as patterns go (at least from what I've seen in open source projects). It doesn't really add much, but might be familiar to some.